docs/tutorials/concepts.rst

```diff
@@ -73,6 +74,37 @@ A policy class typically has the following parts:
 * :meth:`~tianshou.policy.BasePolicy.post_process_fn`: update the buffer with a given batch of data.
 * :meth:`~tianshou.policy.BasePolicy.update`: the main interface for training. This function samples data from buffer, pre-process data (such as computing n-step return), learn with the data, and finally post-process the data (such as updating prioritized replay buffer); in short, ``process_fn -> learn -> post_process_fn``.
 
+
+policy.forward
+^^^^^^^^^^^^^^
+
+The ``forward`` function computes the action over given observations. The input and output is algorithm-specific but generally, the function is a mapping of ``(batch, state, ...) -> batch``.
+
+The input batch is the environment data. The first dimension of all variables in the input ``batch`` should be equal to the batch-size.
```

Comment:
Clarify what are `environment data`. It is not specific enough. It is not clear what it is and from where it comes from.